package.json

```diff
@@ -25,7 +25,8 @@
     "test": "jest --passWithNoTests",
     "test:a11y": "echo 'Axe will run on this command'",
     "test:debug": "node --inspect node_modules/.bin/jest --runInBand",
-    "transpile": "lerna exec --parallel -- babel --root-mode upward src --out-dir lib --source-maps --copy-files"
+    "transpile": "lerna exec --parallel -- babel --root-mode upward src --out-dir lib --source-maps --copy-files",
+    "build-esm": "lerna exec --parallel -- babel --root-mode upward src --config-file ../../babel.config.esm.js --out-dir lib/esm --source-maps --copy-files"
```

Comment:
It uses a different babel configuration file and build into `lib/esm` folder

Reply:
would `yarn transpile --config-file ...` work?